iocore/net/QUICNetVConnection.cc

```diff
@@ -1456,8 +1443,7 @@ void
 QUICNetVConnection::_handle_idle_timeout()
 {
   this->remove_from_active_queue();
-  this->_switch_to_closing_state(std::make_unique<QUICConnectionError>(QUICTransErrorCode::NO_ERROR, "Idle Timeout"));
-  this->handleEvent(QUIC_EVENT_SHUTDOWN, nullptr);
+  this->close(std::make_unique<QUICConnectionError>(QUICTransErrorCode::NO_ERROR, "Idle Timeout"));
```

Comment:
>  A connection enters the draining state  when the idle timeout expires.

Looks incorrect. `close()` will switch state to closing state but not draining state.